rosdep/base.yaml

```diff
@@ -3113,7 +3113,11 @@ libturbojpeg:
   fedora: [turbojpeg-devel]
   freebsd: [libjpeg-turbo]
   gentoo: [media-libs/libjpeg-turbo]
-  ubuntu: [libturbojpeg, libjpeg-turbo8-dev]
+  ubuntu: 
+    '*': [libturbojpeg, libjpeg-turbo8-dev]
+    artful: [libturbojpeg, libjpeg-turbo8-dev, libturbojpeg0-dev]
```

Comment:
wouldn't just listing `libturbojpeg0-dev` be enough ? it looks like it depends on the 2 other packages